discovery/config/config.go

```diff
@@ -15,6 +15,7 @@ package config
 
 import (
 	"github.com/pkg/errors"
+	"github.com/prometheus/prometheus/discovery/cvm"
```

Comment:
please change it to the list below and reorder

Reply:
thanks, fixed.